### PR TITLE
Add persian language

### PR DESCRIPTION
**Pull request by theSinner**
Hi

I added Farsi translations and  Vazir* font to the project and use the font when the language is Farsi to increase the readability. I think adding the option to choose font family is a good idea and if it's okay, I can work on that for my next PR.
Also, I saw an issue with the side nav in RTL mode and fixed that.
Here are some screenshots to show the changes:

### Before:

Screen Shot 2020-10-13 at 12 53 45 AM
Screen Shot 2020-10-13 at 12 53 57 AM
Screen Shot 2020-10-13 at 12 54 16 AM

### After:

Screen Shot 2020-10-13 at 12 46 38 AM
Screen Shot 2020-10-13 at 12 46 51 AM
Screen Shot 2020-10-13 at 12 47 10 AM

* Vazir font is an open-source Farsi font that you can find it in the link below:
https://github.com/rastikerdar/vazir-font

### CODE REVIEW

**Comment by fabiocaccamo** [comment]
Thank you for this great PR, I think that with some small changes it will be perfect.

**Review by fabiocaccamo — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,58 +0,0 @@
...
-# Django stuff:
-*.log
-
-# Sphinx documentation
-docs/_build/
-
-# PyBuilder
-target/
-screenshots
-TODO.txt
```
Comment (merwok) [inline .gitignore:58]: It seems that this file was deleted by mistake!

File: admin_interface/templates/admin/base_site.html [hunk 2]
```diff
@@ -26,12 +27,18 @@
...
 {% include "admin_interface/css/json-widget.css" %}
+{% include "admin_interface/css/rtl.css" %}
 
 {% if theme.css %}
     {{ theme.css|safe }}
 {% endif %}
 
 </style>
+
+{% if current_lang == 'fa' %}
```
[the remark this answers is not in the document]
Comment (merwok) [reply admin_interface/templates/admin/base_site.html:38]: This is specifically a font for Farsi, so why include it for other RTL languages?

File: .gitignore (hunk 1 above)
Comment (fabiocaccamo) [reply .gitignore:58]: ops

File: admin_interface/templates/admin/base_site.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (fabiocaccamo) [reply admin_interface/templates/admin/base_site.html:38]: Ok sorry, I thought that this was a font for all RTL languages, so why don't solve this problem using a font that supports all RTL languages?

[the remark this answers is not in the document]
Comment (theSinner) [reply admin_interface/templates/admin/base_site.html:38]: Yeah, sorry for that. At that time, I just thought about Arabic and Farsi languages and forgot about other RTL languages like Hebrew. Actually, Adobe Arabic font supports multiple languages, but it's not very beautiful and doesn't have good readability in all of them. I can add the font field to the Theme model and the user can choose from installed fonts.

File: admin_interface/locale/fa/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-09-04 09:18+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```
Comment (fabiocaccamo) [inline admin_interface/locale/fa/LC_MESSAGES/django.po:13]: Add your informations.

File: admin_interface/locale/fa/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
```
Comment (fabiocaccamo) [inline admin_interface/locale/fa/LC_MESSAGES/django.po:20]: Could you update all these informations?

File: admin_interface/templates/admin/base_site.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (merwok) [reply admin_interface/templates/admin/base_site.html:38]: Sure, why not.

File: admin_interface/templates/admin_interface/css/rtl.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
+[dir="rtl"] * {
```
[the remark this answers is not in the document]
Comment (merwok) [reply admin_interface/templates/admin_interface/css/rtl.css:1]: Can I ask why?

**Review by fabiocaccamo — APPROVED** [review]
Thank you!

File: .gitignore (hunk 1 above)
Comment (fabiocaccamo) [reply .gitignore:58]: I will add it again after merging.

File: admin_interface/templates/admin/base_site.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (fabiocaccamo) [reply admin_interface/templates/admin/base_site.html:38]: Sorry for the confusion, don't change anything for now, when new needs will arise, we will address them.

File: admin_interface/templates/admin_interface/css/rtl.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (fabiocaccamo) [reply admin_interface/templates/admin_interface/css/rtl.css:1]: Same here, don't change anything.

[the remark this answers is not in the document]
Comment (fabiocaccamo) [reply admin_interface/templates/admin_interface/css/rtl.css:1]: @merwok the reason is that I want to be able to test quickly differences by just removing the `.admin-interface` body class.

**Comment by fabiocaccamo** [comment]
@theSinner @merwok you can upgrade to `0.13.7`.

File: admin_interface/templates/admin_interface/css/rtl.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (merwok) [reply admin_interface/templates/admin_interface/css/rtl.css:1]: Right!  That’s a good reason to add a custom `admin-interface` class, my point was more pointedly about using an `rtl` class vs. an attribute selector.  I see in the commit you made after that you kept the `[dir=rtl]` part so it’s all good!

**Comment by theSinner** [comment]
Thank you for accepting my PR!
@fabiocaccamo

**Comment by fabiocaccamo** [comment]
@theSinner thank you for contributing! 
In `0.14.0` I released sticky list filter feature, could you check RTL compatibilty please?

**Comment by theSinner** [comment]
Of course. I have used this module and the django-colorfield in many projects and I would be happy if I can contribute.
I checked the sticky list filter feature and it works well in RTL mode too.
@fabiocaccamo

**Pull request merged**